docs/UserGuide.md

```diff
@@ -24,6 +24,9 @@ activities but also covers dietary habits, sleep metrics, and more.*
 * Parameters need to be specified in the given order unless specified otherwise.
 * Parameters enclosed in square brackets [] are optional.
 
+**Notes about lack of Goal Delete for Sleep and Activity**
+* Our team will be implementing the delete goal feature for sleep and activity in the next version of AthletiCLI.
+
```

Comment:
I think we need a stronger justification here. It might more likely be a hint for the user to report this feature flaw.

Reply:
Added a new justification for this

1. **Diversity of Diet Goals:** The Diet section likely encompasses a wider range of goals compared to Sleep and Activity. With such variability, users might frequently need to delete diet goals, making a delete function more essential in this section.

2. **Stability of Sleep and Activity Goals:** Goals related to sleep and activity are generally more consistent and less variable over time. This stability reduces the immediate need for a delete feature, as users are less likely to remove these goals frequently.

3. **Focused Development Resources:** Given limited development resources and time, the team prioritized implementing the delete feature for the Diet section, where it was deemed most necessary due to the larger volume and variability of goals.

4. **Planned for Future Implementation:** The absence of this feature in the current version for Sleep and Activity does not indicate it will never be implemented. It is planned for a future update, aligning with a phased development approach.